playbooks/templates/.github/workflows/tft.yml

```diff
@@ -0,0 +1,178 @@
+name: Testing Farm
+on:
+  issue_comment:
+    types:
+      - created
+permissions:
+  contents: read
+  # This is required for the ability to create/update the Pull request status
+  statuses: write
+# The concurrency key is used to prevent multiple workflows from running at the same time
+concurrency:
+{%- raw %}
+  # group name contains reponame-pr_num to allow simualteneous runs in different PRs
+  group: testing-farm-${{ github.event.repository.name }}-${{ github.event.issue.number }}
+  cancel-in-progress: true
+jobs:
+  prepare_vars:
+    name: Get supported platforms from meta/main.yml
+    # Let's schedule tests only on user request. NOT automatically.
+    # Only repository owner or member can schedule tests
+    if: |
```

Comment:
This condition caused the job to be skipped?  https://github.com/linux-system-roles/kernel_settings/actions/runs/10080272663/job/27869614571 - I don't see why